src/nxdoc/nuxeo-server/installation/nuxeo-and-redis.md

```diff
@@ -185,9 +185,9 @@ A Nuxeo No-Redis cluster configuration can be applied at certain conditions, see
 
 The `RedisTransientStore`&nbsp;is a Redis-based implementation of the [Transient Store]({{page page='transient-store'}}).
 
-It becomes the **default** Transient Store if Redis is enabled.
+It is not anymore the **default** Transient Store when Redis is enabled since **LTS2019**, more details in: https://jira.nuxeo.com/browse/NXP-26581.
```

Comment:
Links should be of the form [NXP-26581| https://jira.nuxeo.com/browse/NXP-26581]